TFT/src/User/API/menu.c

```diff
@@ -4,6 +4,7 @@
 #include "Notification.h"
 
 #define STATUS_BAR_REFRESH_TIME 2000  // refresh time in ms
+uint8_t longPressed = 0;
```

Comment:
empty line before this variable definition (separate by empty line `#define` etc. from variable definition etc.)